prisma/schema.prisma

```diff
@@ -61,28 +66,32 @@ model Specialist {
 }
 
 model Organization {
-  id              String             @id @default(uuid()) @db.Uuid
-  createdAt       DateTime           @default(now())
-  name            String             @db.VarChar(128)
-  type            OrganizationType[]
-  yearsOnMarket   Int?               @db.SmallInt
-  formatOfWork    FormatOfWork?
-  addresses       Address[]
-  therapies       Therapy[]
-  isFreeReception Boolean?
-  description     String?            @db.Text
-  phone           String?            @db.VarChar(15)
-  email           String?            @db.VarChar(320)
-  website         String?            @db.Text
-  isActive        Boolean            @default(false)
-  instagram       String?            @db.Text
-  facebook        String?            @db.Text
-  youtube         String?            @db.Text
-  linkedin        String?            @db.Text
-  tiktok          String?            @db.Text
-  viber           String?            @db.Text
-  telegram        String?            @db.Text
-  searchEntry     SearchEntry?
+  id                    String             @id @default(uuid()) @db.Uuid
+  createdAt             DateTime           @default(now())
+  name                  String             @db.VarChar(128)
+  type                  OrganizationType[]
+  ownershipType         OwnershipType?
+  isInclusiveSpace      Boolean?
+  // this is light version of specialists relation to organization
```

Comment:
Can you explain what this comment means?

Reply:
It seems I've created only confusion, which is opposite of what I tried to achieve 😅


The point was to elaborate a bit why organizations could have specializations now, but missing specialists, which may seem a bit confusing